source/main/gui/panels/GUI_VehicleDescription.cpp

```diff
@@ -83,21 +83,36 @@ void VehicleDescription::Draw()
 
             int eventID = RoR::InputEngine::resolveEventName(fmt::format("COMMANDS_{:02d}", i));
             Ogre::String keya = RoR::App::GetInputEngine()->getEventCommand(eventID);
-            eventID = RoR::InputEngine::resolveEventName(fmt::format("COMMANDS_{:02d}", i + 1));
-            Ogre::String keyb = RoR::App::GetInputEngine()->getEventCommand(eventID);
+            Ogre::String keyb;
+
+            for (int x = 1; x < MAX_COMMANDS; x++)
+            {
+                if (!actor->ar_command_key[i].description.empty() && !actor->ar_command_key[i+x].description.empty() &&
+                     actor->ar_command_key[i+x].description == actor->ar_command_key[i].description)
```

Comment:
`i+x` goes outside the array (both i and x range from 1 to MAX_COMMANDS - 1) - you want to use just `x`.